### PR TITLE
Introducing additional documentation

### PR DESCRIPTION
**Pull request by JacobMGEvans**
- Rough draft looking for RFC in PR
- Onboarding may need more details or moved in the layout

### CODE REVIEW

**Comment by JacobMGEvans** [comment]
rocket won't work 😭

File: src/pages/discord/index.mdx [hunk 1]
```diff
@@ -46,3 +52,23 @@ Moderators are responsible for being good examples of the
...
+## Categories
+
+We are always open to more suggestions for additional channels.
+
+- KCD: is where scheduled meetups with Kent occur.
+- General Channels: has a little something for everyone.
+- Epic React: we have channels for helping & discussing every aspect of Epic
+  React 🚀.
+- Testing JavaScript: we have channels for helping & discussing every aspect of
+  Testing JavaScript
```
Comment (kentcdodds) [inline src/pages/discord/index.mdx]:
```suggestion
  Testing JavaScript 🏆
```

**Review by kentcdodds — COMMENTED** [review]
Thanks!

File: src/pages/discord/index.mdx [hunk 2]
```diff
@@ -46,3 +52,23 @@ Moderators are responsible for being good examples of the
...
+We are always open to more suggestions for additional channels.
+
+- KCD: is where scheduled meetups with Kent occur.
+- General Channels: has a little something for everyone.
+- Epic React: we have channels for helping & discussing every aspect of Epic
+  React 🚀.
+- Testing JavaScript: we have channels for helping & discussing every aspect of
+  Testing JavaScript 🏆
+- React: Channels for support and discussion about React, React frameworks, and
+  much more.
```
Comment (kentcdodds) [inline src/pages/discord/index.mdx]: I decided to change the order a bit. I think React makes more sense to come after general.

```suggestion
- General Channels: has a little something for everyone.
- React: Channels for support and discussion about React, React frameworks, and
  much more.
- Epic React: we have channels for helping & discussing every aspect of Epic
  React 🚀.
- Testing JavaScript: we have channels for helping & discussing every aspect of
  Testing JavaScript 🏆
```

**Review by kentcdodds — APPROVED** [review]
This is great 👍

**Comment by kentcdodds** [comment]
This is a nice improvement 👏

**Pull request merged**